driver/conv_driver.hpp

```diff
@@ -1394,8 +1510,11 @@ int ConvDriver<Tgpu, Tref>::AllocateBuffersAndCopy()
             db_host     = tensor<Tref>(miopen::deref(biasTensor));
             for(int i = 0; i < b_sz; i++)
             {
-                b.data[i] = static_cast<Tgpu>(i % 8) + prng::gen_canonical<Tgpu>();
-                db[i]     = static_cast<Tgpu>(i % 8) + prng::gen_canonical<Tgpu>();
+                b.data[i] =
+                    static_cast<Tgpu>(i % 8) +
+                    (is_fp8 ? prng::gen_A_to_B(Data_min, Data_max) : prng::gen_canonical<Tgpu>());
```

Comment:
[R] if Data_min/max = 0/1 for non-F8 types, then 
```suggestion
                    static_cast<Tgpu>(i % 8) + prng::gen_A_to_B(Data_min, Data_max);
```
as `gen_canonical<T>()` is equivalent to `gen_A_to_B(static_cast<T>(0), static_cast<T>(1))`

@CAHEK7 right?